syncer/dml.go

```diff
@@ -130,6 +147,26 @@ func genUpdateSQLs(param *genDMLParam) ([]string, [][]string, [][]interface{}, e
 			oriChangedValues = extractValueFromData(oriChangedData, ti.Columns)
 		}
 
+		for _, expr := range oldValueFilters {
```

Comment:
Does user expected `old-value-filter` and `new-value-filter` to be `and` relation instead of `or` relation?  Or what if user want `and` but we only support `or`

Reply:
How about we add a field in config, to let user specify "AND" "OR"

Reply:
I think we can change the `UpdateFilter` to include both old and new filter, if user only want to change one of them then just leave the other empty, thus the semantic  is always `AND`. For the `OR` case, user can just add two separated filter

Reply:
OK.

We might also make (insert-value-expr), (update-old-value-expr, update-new-value-expr), (delete-value-expr) these three groups exclusive, to indicate AND semantic.